tracking_manager/views/ir_model.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!-- Copyright (C) 2022 Akretion (<http://www.akretion.com>).
+     @author Kévin Roche <kevin.roche@akretion.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_model_form" model="ir.ui.view">
+        <field name="name">tracking.ir.model form</field>
+        <field name="model">ir.model</field>
+        <field name="inherit_id" ref="base.view_model_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//sheet/group[1]" position="after">
+
+                <group string="Custom Tracking">
+                    <group>
+                        <field name="active_custom_tracking" string="Active" />
+                        <field
+                            name="automatic_custom_tracking"
+                            string="Automatic configuration"
+                            invisible="not active_custom_tracking"
+                        />
+                        <field
+                            name="automatic_custom_tracking_domain"
+                            string="Domain"
+                            invisible="not automatic_custom_tracking"
+                            required="automatic_custom_tracking"
+                        />
```

Comment:
```suggestion
                        <field
                            name="automatic_custom_tracking_domain"
                            string="Domain"
                            invisible="not automatic_custom_tracking"
                            required="automatic_custom_tracking"
                            widget="domain"
                            options="{'model': 'ir.model.fields', 'in_dialog': True, 'foldable': True}"
                        />
```

Reply:
I totally agree, but the record result is a little misleading, because the related table is not filtered. I have the same issue in one of our own customizations. Do you have an idea how we could improve this?
![image](https://github.com/OCA/server-tools/assets/1799080/61204094-06af-47d1-8cce-00a0ba641329)

Reply:
ok solved it myself.
Thank you for all your input. I always apricate review input.

Reply:
I think it looks better like this then you can actually visualize the fields you will be tracking if you update before actually updating, it is more user friendly IMHO.